src/Symfony/Component/Serializer/Tests/Normalizer/DateIntervalNormalizerTest.php

```diff
@@ -119,6 +119,22 @@ public function testDenormalizeIntervalsWithOmittedPartsBeingZero()
         $this->assertDateIntervalEquals($this->getInterval('P0Y0M0DT12H34M0S'), $normalizer->denormalize('PT12H34M', \DateInterval::class));
     }
 
+    /**
+     * Since PHP 8.0 DateInterval::construct supports periods containing both D and W period designators.
+     *
+     * @requires PHP 8
+     *
+     * @testWith ["%rP%yY%mM%wW%dDT%hH%iM%sS","P3W2D", 23]
```

Comment:
As you have only one dataset, I think `@testWith` is not needed

Reply:
fixed